apps/els_lsp/src/els_bsp_client.erl

```diff
@@ -40,14 +42,15 @@
 %% Defines
 %%==============================================================================
 -define(SERVER, ?MODULE).
--define(TIMEOUT, 10000).
+-define(BSP_WILDCARD, "*.json").
+-define(BSP_CONF_DIR, ".bsp").
```

Comment:
Is this convention? So there is a 

```
.bsp/
```
with some json files which look like
```
{language: ["erlang"], argv = ["rebar3", "...", ...]}
```

Reply:
The _Build Server Discovery_ mechanisim and format are documented [here](https://build-server-protocol.github.io/docs/server-discovery.html#the-bsp-connection-details).